src/module-1/cutStrings/index.js

```diff
@@ -1 +1,12 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = []) => {
+  if (arr.length > 0) {
+    let shortestWord = arr[0];
```

Comment:
consider having `minLength = 0` instead of `shortestWord`, for you calculate lengths in `.forEach` already, let's use them;  
please use `Math.min()` method in `.forEach` loop;  
please use short return form: `() => val` for `.map` method